src/components/Calculator.js

```diff
@@ -18,14 +19,16 @@ class Calculator extends React.Component {
 
   render() {
     return (
-      <CalcWrapper>
-        <CalcScreen value="0" />
-        <ButtonsPad>
-          {this.btnValues.flat().map((row, index) => (
-            <Button key={index} value={row} />
-          ))}
-        </ButtonsPad>
-      </CalcWrapper>
+      <CalcProvider>
+        <CalcWrapper>
+          <CalcScreen value="0" />
+          <ButtonsPad>
+            {this.btnValues.flat().map((row, index) => (
+              <Button key={index} value={row} />
+            ))}
+          </ButtonsPad>
```

Comment:
- [ ] You need to use `state` and `props` and make that props get passed to child components of the `Calculate.js` component.

- I know you can do it 💪🏻 Great job 👍🏻

Reply:
Thanks Mr. @BenMukebo  for the great feedback. I have already implemented the requested changes.